packages/mobile/src/pincode/authentication.ts

```diff
@@ -230,3 +231,12 @@ export async function ensureCorrectPassword(
     return false
   }
 }
+
+export async function removeAccountLocally() {
+  clearPasswordCaches()
+  const removed = await removeStoredItem(STORAGE_KEYS.PEPPER)
+  if (!removed) {
+    // This shouldn't happen, should we do something here?
+  }
+  // TODO: Remove the account from geth.
```

Comment:
- Is there anything else we should do when deleting the account? 
- Right now we are not deleting the account from geth because there is no method in contractkit to do so. I'm not sure it's the same thing, but there's an open issue that might be related to this: https://github.com/celo-org/celo-monorepo/issues/4359

Reply:
Looks like we should also remove the `passwordHash`, see https://github.com/celo-org/celo-monorepo/blob/1ca65c95495f91138e1538618bd24865ece6737e/packages/mobile/src/pincode/authentication.ts#L105-L106

Reply:
We should also reset the analytics session (flush current events and reset for a new identity).

Reply:
Good catch, thanks!

Reply:
Just thought about the anonymous firebase session we have should be closed too.